src/NServiceBus.Persistence.CosmosDB/Transaction/TransactionInformationConfiguration.cs

```diff
@@ -36,12 +36,29 @@ public class TransactionInformationConfiguration
         /// <param name="extractor">The extraction function.</param>
         /// <param name="extractorArgument">The argument passed as state to the <paramref name="extractor"/></param>
         /// <typeparam name="TMessage">The message type to match against.</typeparam>
-        /// <typeparam name="TArg">The argument type.</typeparam>
+        /// <typeparam name="TArg">The argument passed as state to the <paramref name="extractor"/></typeparam>
```

Comment:
Unfortunately, this is really the type and not the actual state passed. The actual state passed is represented by the `extractorArgument`